keel-api-jackson/src/main/kotlin/com/netflix/spinnaker/keel/json/mixins/LocatableMixin.kt

```diff
@@ -0,0 +1,9 @@
+package com.netflix.spinnaker.keel.json.mixins
+
+import com.fasterxml.jackson.annotation.JacksonInject
+import com.netflix.spinnaker.keel.api.Locations
+
+interface LocatableMixin<T : Locations<*>> {
```

Comment:
Side-comment: I understand the reason, but having to do all this in code as opposed to using annotations feels like a step backwards... 🙁 What's the issue with annotations and plugins?